### PR TITLE
chore upgrade @fluentui/react

### PR DESCRIPTION
**Pull request by devinea**

### CODE REVIEW

**Review by 815are — COMMENTED** [review]
do not see much problems, currently noticed following small issue:
1. new ->
image

2. old ->
image

**Review by 815are — DISMISSED** [review]
checked:
1. tested with storybook - small fix was applied to dialog to center footer actions, otherwise did not notice issues
2. changeset is presented

**Comment by devinea** [comment]
@815are thanks for the review and updates.

**Review by marufrasully — DISMISSED** [review]
Thanks @devinea 
* Changes looks good
* Did not test it locally

File: packages/ui-components/src/components/UIContextualMenu/UIContextualMenu.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type {
     ICalloutContentStyles,
     IRawStyle,
     IStyleFunctionOrObject,
-    ICalloutContentStyleProps
+    IContextualMenuStyleProps
```
Comment (devinea) [inline packages/ui-components/src/components/UIContextualMenu/UIContextualMenu.tsx:8]: @Jimmy-Joseph19 Is this OK?

Comment (Jimmy-Joseph19) [reply packages/ui-components/src/components/UIContextualMenu/UIContextualMenu.tsx:8]: This changes looks fine. Sorry for delay in response

**Review by 815are — APPROVED** [review]
reapproving

**Comment by devinea** [comment]
Thanks all

**Pull request merged**